allen-cell-animated/cell-catalog
# render subpage with some values

Problem
=======
we want to display the cell line data in the subpage as shown [in the design ](https://www.figma.com/design/kMIJN6BoOkXIhnVUruIHNT/Website-allencell.org?node-id=1198-6040&t=2FDkMryTqL7sbPnJ-0)
closes #40 

Solution
========
Unpacked the data and rendered it in a card. This involved a lot of color updates as well. The design was updated with a [new color palette](https://www.figma.com/design/kMIJN6BoOkXIhnVUruIHNT/Website-allencell.org?node-id=2420-11402&t=2FDkMryTqL7sbPnJ-0), so the colors for the whole app are change to be consistent with the naming scheme of the colors. 

with @rugeli @tyler-foster and @interim17 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. npm start 
2. got to http://localhost:8000/disease-cell-line/AICS-97/
3. should look like the design 

Or:
https://deploy-preview-38--cell-catalog.netlify.app/disease-cell-line/AICS-97/


Screenshots (optional):
-----------------------
<img width="547" alt="Screenshot 2024-08-27 at 3 34 21 PM" src="https://github.com/user-attachments/assets/8cac31fc-569e-4b9a-8f9a-36daddd2142f">


Keyfiles (delete if not relevant):
-----------------------
https://github.com/allen-cell-animated/cell-catalog/pull/38/files#diff-82cc1abec44f9670a1da427352210dbe4c718168dd8139139810debcb06864d2
